designsafe/apps/workspace/models/app_entries.py

```diff
@@ -4,26 +4,35 @@
 from django.db import models
 
 APP_ICONS = [
+    ("Generic-App", "Generic: Application"),
+    ("Generic-Vis", "Generic: Visualization"),
+    ("Earth", "Element: Earth"),
+    ("Water", "Element: Water"),
+    ("Wind", "Element: Wind"),
+    ("All-Hazards", "All Hazards"),
     ("adcirc", "ADCIRC"),
-    ("ansys", "Ansys"),
-    ("blender", "Blender"),
-    ("clawpack", "Clawpack"),
-    ("compress", "Compress"),
-    ("dakota", "Dakota"),
-    ("extract", "Extract"),
-    ("hazmapper", "Hazmapper"),
-    ("jupyter", "Jupyter"),
-    ("ls-dyna", "LS-DYNA"),
-    ("matlab", "MATLAB"),
-    ("ngl", "NGL"),
-    ("openfoam", "OpenFOAM"),
-    ("opensees", "OpenSees"),
-    ("paraview", "Paraview"),
-    ("qgis", "QGIS"),
-    ("rwhale", "rWHALE"),
-    ("stko", "STKO"),
-    ("swbatch", "swbatch"),
-    ("visit", "VisIt"),
```

Comment:
🧐 Is this being deleted? I don't see a new instance of it.

Reply:
Ah… Hm, I found it as Water, so never mind, perhaps.

Reply:
VisIt is going to become `Generic-Vis`